examples/JsonPub/JsonPub.ino

```diff
@@ -1,17 +1,13 @@
-// A simple example demonstrating periodic JSON publications (with a non-blocking loop())
 #include <Redis.h>
 #include <ArduinoJson.h>
 
-// set to your WiFi information
-#define WIFI_SSID       ""
-#define WIFI_PASSWORD   ""
+#define WIFI_SSID       "Safado2GH"
+#define WIFI_PASSWORD   "Summer2017"
 
-// set to your Redis server information
-#define REDIS_ADDR      "0.0.0.0"
-#define REDIS_PORT      6379
-#define REDIS_PASSWORD  ""
+#define REDIS_ADDR      "redis-17772.c60.us-west-1-2.ec2.cloud.redislabs.com"
+#define REDIS_PORT      17772
+#define REDIS_PASSWORD  "qsBmsA51hnjNkfpQBIDE8gq6dJeTBqFc"
```

Comment:
Please go ahead and revert these to the original, generic values: I'm guessing you didn't want your server's password checked in anyway (nor your WiFi password for that matter)! :wink: